generate-api/src/generator.rs

```diff
@@ -491,29 +491,86 @@ impl CodeGenerator {
             f,
             r#"
 
+            /// Locales matching the locales in `glibc`.
+            ///
+            /// Most locale names follow the syntax `language[_territory][@modifier]`.
+            /// The `@` is replaced with `_` in the `enum` variant names.
+            ///
+            /// The default locale is `POSIX`.
+            ///
+            /// License note: The Free Software Foundation does not claim any copyright interest in the locale
+            /// data of the GNU C Library; they believe it is not copyrightable.
```

Comment:
This license note seems quite out of place :smile:.
When a someone browses the documentation of an MIT-licensed library and then seeds "glibc" and "GNU" I expect this will be a question.

Reply:
Good to see the complete context, https://github.com/chronotope/chrono/pull/453#issue-655205860 had the important bit of it :+1:.